### PR TITLE
removes sexchem jumpsuit ripping

### PR DESCRIPTION
**Pull request by silicons**
## Why It's Good For The Game

this was funny for a while but now it's just banbait and stupid no one wants to see your K cup breasts out in public no one wants to see 36 inch dicks every round this was funny until it wasn't and it's time to take this out back

## Changelog
:cl: silicons
del: sexchem breast/penis enlargement no longer prevents you from wearing clothes
/:cl:

### CODE REVIEW

**Comment by Detective-Google** [comment]
pogchamp

**Comment by ghost** [comment]
where have i seen this before

File: modular_citadel/code/datums/status_effects/chems.dm [hunk 1]
```diff
@@ -41,20 +41,6 @@
 
 /datum/status_effect/chem/breast_enlarger/on_apply()//Removes clothes, they're too small to contain you. You belong to space now.
 	log_reagent("FERMICHEM: [owner]'s breasts has reached comical sizes. ID: [owner.key]")
```
Comment (LetterN) [inline modular_citadel/code/datums/status_effects/chems.dm:43]: does this spam log?

**Review by Ghommie — APPROVED** [review]
no comment.

**Pull request merged**